jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/README.md

```diff
@@ -0,0 +1,11 @@
+# CLI for JFrog Platform Services
+
+This page describes how to use JFrog CLI with JFrog Platform Services.
```

Comment:
I'm not sure that the term "Platform Services" would be clear for the user. You can consult with s technical writer about the best introductory phrasing. 
There's a chance some of the phrasing throughout this PR can be further improved.